centos2ol.sh

```diff
@@ -486,6 +486,23 @@ case "$os_version" in
         ;;
 esac
 
+echo "Testing for remaining CentOS RPMs"
+# If CentOS and Oracle Linux have identically versioned RPMs then those RPMs are left unchanged.
+#  This should have no technical impact but for completeness, reinstall these RPMs
+#  so there is no accidental cross pollination.
+mapfile -t list_of_centos_rpms < <(rpm -qa --qf "%{NAME}-%{VERSION}-%{RELEASE} %{VENDOR}\n" | grep CentOS | awk '{print $1}')
+if [[ -n "${list_of_centos_rpms[*]}" ]]; then
+    echo "Reinstalling RPMs: ${list_of_centos_rpms[*]}"
+    yum -y reinstall "${list_of_centos_rpms[@]}"
```

Comment:
Do you think this is too aggressive @Djelibeybi ?
I'm not sure what else CentOS sign and unsure how much to soften the blow. 
* We could limit it to reinstalling just hose in `ol*` repos but that means downloading the metadata of all repos (200+MB today on OL7)
* We could limit it to reinstalling just those in `ol?_latest` repos but that may be too narrow

Reply:
Why not make it someone else's problem and enable it via a command-line parameter, e.g. `--replace-all`?

Reply:
Good idea, I'll default to just `ol?_latest` repo unless that argument is passed

Reply:
Then document that this is useful/necessary if user wants to remove the CentOS GPG key. But I would also point out that it is not required to be eligible for support from Oracle.

Reply:
Honestly, I'd just stop with the distro-sync unless this is passed. It adds a lot of download and time to the process.

Reply:
Sure, in that case I won't print the non-Oracle RPMs if this option isn't selected as it loses value.

Reply:
Agreed.